Makefile

```diff
@@ -73,67 +74,59 @@ generate-deepcopy:
 	--bounding-dirs "github.com/atlassian/smith/pkg/apis/smith/v1,github.com/atlassian/smith/examples/sleeper/pkg/apis/sleeper/v1" \
 	--output-file-base zz_generated.deepcopy
 
-.PHONY: minikube-test
-minikube-test: fmt update-bazel
+.PHONY: integration-test
+integration-test: fmt update-bazel
 	bazel test \
 		--test_env=KUBE_PATCH_CONVERSION_DETECTOR=true \
 		--test_env=KUBE_CACHE_MUTATION_DETECTOR=true \
-		--test_env=KUBERNETES_SERVICE_HOST="$$(minikube ip)" \
-		--test_env=KUBERNETES_SERVICE_PORT=8443 \
-		--test_env=KUBERNETES_CA_PATH="$$HOME/.minikube/ca.crt" \
-		--test_env=KUBERNETES_CLIENT_CERT="$$HOME/.minikube/apiserver.crt" \
-		--test_env=KUBERNETES_CLIENT_KEY="$$HOME/.minikube/apiserver.key" \
+		--test_env=KUBERNETES_CONFIG_FROM=file \
```

Comment:
if this is a switch that allows loading from a different place, sometimes, it's probably worth having a test that checks you _can_ still load from envvars.